src/app/home/home.page.ts

```diff
@@ -32,6 +32,7 @@ export class HomePage {
   @ViewChild('bannerTemplate') bannerTemplate!: TemplateRef<any>;
   @ViewChild('solutionTemplate') solutionTemplate!: TemplateRef<any>;
   @ViewChild('recommendationTemplate') recommendationTemplate!: TemplateRef<any>;
+  sharePopupHandler:any;
```

Comment:
Give relevant names based on the action done
Its not a sharePopupHandler

Reply:
done